torch/csrc/stable/tensor.h

```diff
@@ -109,6 +109,20 @@ class Tensor {
     return device_type == aoti_torch_device_type_cuda();
   }
 
+  bool is_cpu() const {
+    int32_t device_type;
+    TORCH_ERROR_CODE_CHECK(
+        aoti_torch_get_device_type(ath_.get(), &device_type));
+    return device_type == aoti_torch_device_type_cpu();
+  }
+
+  int32_t dtype() const {
```

Comment:
Document what this actually does, because it returns an int32_t that doesn't mean anything for the user. This is the reason we never added dtype, as every API in this code should parallel with the APIs in TensorBase.h.

What is the pro of having this wrapper?